include/z64save.h

```diff
@@ -273,6 +273,13 @@ typedef enum SunsSongState {
     /* 3 */ SUNSSONG_SPECIAL // time does not advance, but signals the song was played. used for freezing redeads
 } SunsSongState;
 
+typedef enum DungeonIndex {
+    /* 0 */ DUNGEON_INDEX_WOODFALL,
+    /* 1 */ DUNGEON_INDEX_SNOWHEAD,
+    /* 2 */ DUNGEON_INDEX_GREAT_BAY,
+    /* 3 */ DUNGEON_INDEX_STONE_TOWER,
```

Comment:
I'd note that this also applies to Inverted Stone Tower Temple even though it's a different scene. We should also distinguish between Stone Tower (the outside) and the Temple itself.

Reply:
I thought just labeling it "**Dungeon** Index" was sufficient, no? Stone Tower is not a dungeon, whereas Stone Tower Temple is.

Reply:
Yeah, I'm thinking more generally, what you get if you search for one or the other.

Reply:
Hmm, then should they *all* have TEMPLE at the end? I don't see how that same line of logic doesn't apply for Woodfall, for example. I still see this as unnecessary verbosity, but I could do it for all of them.

Reply:
Actually it's a fair point for all of them, come to think of it. I think we can manage without, then. One could consider whether we should call them temples instead of dungeons thoughout, but it's probably not worth it.

Reply:
Whoops, well, I already added TEMPLE, expecting you to say that was a good idea, lol. Let me know if I should remove it.

Reply:
I'm not that bothered until it becomes obvious that it's stupidly long in use.

Reply:
Sure, let's just keep TEMPLE for now then and see how it gets used. Can always drop it later.